Formula/log4cplus.rb

```diff
@@ -3,8 +3,7 @@ class Log4cplus < Formula
   homepage "https://sourceforge.net/p/log4cplus/wiki/Home/"
   url "https://downloads.sourceforge.net/project/log4cplus/log4cplus-stable/2.0.5/log4cplus-2.0.5.tar.xz"
   sha256 "6046f0867ce4734f298418c7b7db0d35c27403090bb751d98e6e76aa4935f1af"
-  # license ["Apache-2.0", "BSD-2-Clause"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "Apache-2.0"
+  license any_of: ["Apache-2.0", "BSD-2-Clause"]
```

Comment:
```suggestion
  license all_of: ["Apache-2.0", "BSD-2-Clause"]
```

From the `LICENSE` file:

> Each file of log4cplus source is licensed using either two clause BSD
> license or Apache license 2.0. Log4cplus is derived work from log4j.

I interpret that as saying some file are under `Apache-2.0` and some are under `BSD-2-Clause`, so we must include both.

Reply:
Hmm, that's a tricky one. It's not clear to me whether they're saying that each file is exclusively licensed under one of those licenses, or that each file can be considered to be licensed under either license. I'll wait for others to comment before making any changes here.

Reply:
Looking at individual files, they each have one of the two licenses. So it's `all_of:`